### ISSUE TITLE
Strangely named library files mean find_package OpenSSL doesn't work

### ISSUE THREAD

**Issue by NeilChatterjee** [janbar/openssl-cmake#14]
I notices that because this builds OpenSSL with library names like:

`crypto_1_1x`

instead of `crypto`

it means if you build/install OpenSSL with this repo, it won't be findable via find_package(OpenSSL)

So I made a little flag in my fork to prevent that renaming. It's called `OPENSSL_BUILD_WITH_INSTALL_NAMES`

I'm sending a PR if you want to take it!

**Comment by NeilChatterjee** [comment]
https://github.com/janbar/openssl-cmake/pull/15

**Comment by janbar** [comment]
Yep, the renaming is effective on Windows platform to be conform with Qt5 usage.
